mock-caas/src/test/java/com/sequenceiq/caas/grpc/service/MockUserManagementServiceTest.java

```diff
@@ -61,4 +65,69 @@ public void testCreateWorkloadUsername() {
 
         Assert.assertEquals(expected, underTest.sanitizeWorkloadUsername(username));
     }
+
+    @Test
+    public void testgetActorWorkloadCredentials() {
```

Comment:
this test seems superfluous since it just duplicates exactly the code in the implementation

Reply:
True, I have made it shorter.